### PR TITLE
add skeleton component displays the raw full case as json

### PR DESCRIPTION
**Pull request by attwad**

### CODE REVIEW

File: verification/curator-service/ui/src/components/LinelistTable.test.tsx [hunk 1]
```diff
@@ -329,6 +329,66 @@ it('can go to page to edit a row', async () => {
...
+        <Router history={history}>
+            <LinelistTable user={curator} />
+        </Router>,
+    );
+    expect(mockedAxios.get).toHaveBeenCalledTimes(1);
+    expect(mockedAxios.get).toHaveBeenCalledWith('/api/cases/?limit=10&page=1');
+    const row = await findByText('some notes');
+    expect(row).toBeInTheDocument();
+
+    const editButton = getByText(/details/);
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/LinelistTable.test.tsx:387]: s/editButton/detailsButton

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 2]
```diff
@@ -287,29 +287,39 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
...
-                                          // Somehow the templating system doesn't think row has an id property but it has.
-                                          const id = (row as TableRow).id;
-                                          history.push(`/cases/edit/${id}`);
-                                      },
+                              },
+                              {
+                                  icon: 'edit',
+                                  tooltip: 'Edit this case',
+                                  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                                  onClick: (_: any, row: any): void => {
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:304]: If you make this `row: TableRow` can you get rid of the cast below?

File: verification/curator-service/ui/src/components/LinelistTable.test.tsx (hunk 1 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.test.tsx:387]: done.

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 2 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:304]: no and that's what's weird about this :(

**Comment by attwad** [comment]
[Page to display a full case details](https://trello.com/c/OcIt0g8b/216-page-to-display-a-full-case-details)

**Pull request merged**